src/nl/Stratego/Bord.java

```diff
@@ -69,28 +80,27 @@ public void move(int x, int y, int x_new, int y_new){
     public String toString(){
         StringBuilder bordstring = new StringBuilder();
         bordstring.append("+---+---+---+---+---+---+---+---+---+---+\n");
-        for (int x = 0; x < 10; x++){
-            for (int y = 0; y < 10; y++){ //deze forloop voegt voor ieder vakje de value van het spelstuk toe of een "o" als het vakje leeg is.
-                String spelstukString;
-                if (SpeelStukken[x][y] instanceof Speelstuk){
-                    Speelstuk speelstuk = (Speelstuk)SpeelStukken[x][y];
-                    int value = speelstuk.getValue();
-                    if (value < 10){
-                        spelstukString = "|  " + value; //een extra spatie toevoegen als de waarde kleiner is dan tien, zodat de uitlijning mooi klopt.
-                    } else {
-                        spelstukString = "| " + value;
+            for (int y = 0; y < 10; y++) {
+                for (int x = 0; x < 10; x++) { //deze forloop voegt voor ieder vakje de value van het spelstuk toe of een "o" als het vakje leeg is.
+                    String spelstukString;
+                    if (SpeelStukken[y][x] instanceof Speelstuk) {
+                        Speelstuk speelstuk = (Speelstuk) SpeelStukken[y][x];
+                        int value = speelstuk.getValue();
+                        if (value < 10) {
+                            spelstukString = "|  " + value; //een extra spatie toevoegen als de waarde kleiner is dan tien, zodat de uitlijning mooi klopt.
+                        } else {
+                            spelstukString = "| " + value;
+                        }
+                    } else if (SpeelStukken[y][x] instanceof String) { //als er niet naar een object wordt verwezen op deze plek is deze plek leeg
+                        spelstukString = "| x ";
+                    } else { // en anders is het een muur.
```

Comment:
oh de comments hier kloppen dus niet wanneer het een muur is of een open plek... Oops